README.md

```diff
@@ -1,2 +1,66 @@
-# ruby_school_library
-School library is a fully OOP library managment system built using ruby as a console application. The user has the ability to add books, persons and create rentals for specific books.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# School library
+
+> School library is a fully OOP library managment system built using ruby as a console application. The user has the ability to add books, persons and create rentals for specific books.
+
+## Live Demo
+
+> Comming soon⏳
+
+## Built With
+
+- Ruby
+
+### Technologies used
+
+- Git, Github.
+- Rubocop
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- use git clone in order to get the copy this repo. You need to have atleast basic understanding of Ruby.
+
+## Setup
+
+To get a local copy up and running you'll need to have Ruby installed on your local machine.
+
+### Install
+
+After installing Ruby please follow the next steps...
+
+### Usage
+
+-- To clone this project use:
+
+```bash
+
+git clone git@github.com:uisendy/ruby_school_library.git
+
+```
+
+## Author
+
+👤 **Sendy Inieke**
+
+- GitHub: [@uisendy](https://github.com/uisendy)
+
+- LinkedIn: [sendyinieke](https://www.linkedin.com/in/sendyinieke/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
Great job @uisendy 🤝

- [ ] [Optional] Kindly fix the MIT license link. As for now, it appears to be broken.